### PR TITLE
SDC fixed Instrumented Android unit tests

### PR DESCRIPTION
**Pull request by kashyapj2793**
Addressing issue #149

### CODE REVIEW

File: datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactoryInstrumentedTest.kt [hunk 1]
```diff
@@ -18,21 +18,33 @@ package com.google.android.fhir.datacapture.views
...
-  private val parent = FrameLayout(InstrumentationRegistry.getInstrumentation().context)
-  private val viewHolder = QuestionnaireItemDatePickerViewHolderFactory.create(parent)
+  private lateinit var context: ContextThemeWrapper
+  private lateinit var parent: FrameLayout
+  private lateinit var viewHolder: QuestionnaireItemViewHolder
+
+  @Before
+  fun setUp() {
+    context = ContextThemeWrapper(InstrumentationRegistry.getInstrumentation().getTargetContext(),
+      R.style.Theme_MaterialComponents)
```
Comment (jingtang10) [inline datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactoryInstrumentedTest.kt:44]: nit: indentation

```
context = ContextThemeWrapper(
    InstrumentationRegistry.getInstrumentation().getTargetContext(),
    R.style.Theme_MaterialComponents
)
```

File: datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemEditTextViewHolderFactoryInstrumentedTest.kt [hunk 2]
```diff
@@ -16,24 +16,34 @@
...
 class QuestionnaireItemEditTextViewHolderFactoryInstrumentedTest {
-  private val parent = FrameLayout(InstrumentationRegistry.getInstrumentation().context)
-  private val viewHolder = QuestionnaireItemEditTextViewHolderFactory.create(parent)
+  private lateinit var context: ContextThemeWrapper
+  private lateinit var parent: FrameLayout
+  private lateinit var viewHolder: QuestionnaireItemViewHolder
+
+  @Before
+  fun setUp() {
+    context = ContextThemeWrapper(InstrumentationRegistry.getInstrumentation().getTargetContext(),
```
Comment (jingtang10) [inline datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemEditTextViewHolderFactoryInstrumentedTest.kt:42]: ditto

File: datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemGroupViewHolderFactoryInstrumentedTest.kt [hunk 3]
```diff
@@ -18,19 +18,30 @@ package com.google.android.fhir.datacapture.views
...
 class QuestionnaireItemGroupViewHolderFactoryInstrumentedTest {
-  private val parent = FrameLayout(InstrumentationRegistry.getInstrumentation().context)
-  private val viewHolder = QuestionnaireItemGroupViewHolderFactory.create(parent)
+  private lateinit var context: ContextThemeWrapper
+  private lateinit var parent: FrameLayout
+  private lateinit var viewHolder: QuestionnaireItemViewHolder
+
+  @Before
+  fun setUp() {
+    context = ContextThemeWrapper(InstrumentationRegistry.getInstrumentation().getTargetContext(),
```
Comment (jingtang10) [inline datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemGroupViewHolderFactoryInstrumentedTest.kt:40]: ditto

**Review by jingtang10 — APPROVED** [review]
just some nits about indentation.

thanks for fixing 👍

File: datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactoryInstrumentedTest.kt [hunk 4]
```diff
@@ -18,21 +18,34 @@ package com.google.android.fhir.datacapture.views
...
-  private val viewHolder = QuestionnaireItemDatePickerViewHolderFactory.create(parent)
+  private lateinit var context: ContextThemeWrapper
+  private lateinit var parent: FrameLayout
+  private lateinit var viewHolder: QuestionnaireItemViewHolder
+
+  @Before
+  fun setUp() {
+    context = ContextThemeWrapper(
+      InstrumentationRegistry.getInstrumentation().getTargetContext(),
+      R.style.Theme_MaterialComponents)
```
Comment (jingtang10) [inline datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactoryInstrumentedTest.kt:45]: nit: new line for the closing bracket:
```
context = ContextThemeWrapper(
      InstrumentationRegistry.getInstrumentation().getTargetContext(),
      R.style.Theme_MaterialComponents
)
```

Comment (kashyapj2793) [reply datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactoryInstrumentedTest.kt]: yes sorry I missed that

**Comment by jingtang10** [comment]
apologies, after reading https://kotlinlang.org/docs/reference/coding-conventions.html#formatting i realise the kotlin styleguide advocates for 4 whitespaces... i will update my IDE settings.

**Comment by kashyapj2793** [comment]
No problem I'll update it in my IDE too. I'll merge this PR after the checks are complete.

**Pull request merged**